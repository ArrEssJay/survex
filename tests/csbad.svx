; pos=fail warn=0 error=23
*fix 1 reference 0 0 0
*cs EPSG:-1
*cs ERSI:1234
*cs EUR79Z31
*cs IJTSK 03
*cs IJTSK04
*cs IJTSK99
*cs IJTSK3
*cs IJTSK032
*cs IJTSK03a
*cs lat-lat
*cs Long-LONG
*cs OSGB:ZZ
*cs OSGB:HI
*cs S-LONG
*cs UTM0
*cs UTM61N
*cs UTM61S
*cs custom "+proj=ection"
*cs out lat-long
*cs out long-lat
*cs out jtsk
*cs out jtsk03
