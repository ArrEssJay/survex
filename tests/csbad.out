./csbad.svx:1: Station "1" fixed before CS command first used
./csbad.svx:2:4: Unknown coordinate system
./csbad.svx:3:4: Unknown coordinate system
./csbad.svx:4:4: Unknown coordinate system
./csbad.svx:5:11: End of line not blank
./csbad.svx:6:4: Unknown coordinate system
./csbad.svx:7:4: Unknown coordinate system
./csbad.svx:8:4: Unknown coordinate system
./csbad.svx:9:4: Unknown coordinate system
./csbad.svx:10:4: Unknown coordinate system
./csbad.svx:11:4: Unknown coordinate system
./csbad.svx:12:4: Unknown coordinate system
./csbad.svx:13:4: Unknown coordinate system
./csbad.svx:14:4: Unknown coordinate system
./csbad.svx:15:4: Unknown coordinate system
./csbad.svx:16:4: Unknown coordinate system
./csbad.svx:17:4: Unknown coordinate system
./csbad.svx:18:4: Invalid coordinate system: unknown projection id
./csbad.svx:19:4: Unknown coordinate system
./csbad.svx:20:4: Coordinate system unsuitable for output
./csbad.svx:21:4: Unknown coordinate system
./csbad.svx:22:4: Unknown coordinate system

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
Done.
There were 0 warning(s) and 22 error(s) - no output files produced.
